travis-ci.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+set +e
+
+if [ -z "$TRAVIS" ]; then
+    echo "this script is intended to be run only on travis" >&2
+    exit 1
+fi
+
+KUBECTL_WAIT_CMD="kubectl wait --timeout=5m --for=condition=Ready pod/sonobuoy -n heptio-sonobuoy"
+SONOBUOY_CMD=./sonobuoy
+SONOBUOY_SUB_CMD=run -m Quick
+
+function waitForDeployment() {
+  echo "|---- Waiting for deployment to be 'Ready'..."
+
+  # Attempt get sonobuoy pod for readiness
+  $KUBECTL_WAIT_CMD
+  while [ $? -eq 0 ] 
+  do
+    if [ $? -eq 0 ]
+    then
+      break
+    else
+      echo "sonobuoy not ready..."
+      sleep 2
+      # Attempt get sonobuoy pod for readiness
+      $KUBECTL_WAIT_CMD
+    fi
+  done
+}
+
+function checkForResults() {
+  echo "|---- Looking for results..."
+
+  # Attempt to get status
+  $SONOBUOY_CMD retrieve
```

Comment:
So the fear as of now is that you can get a `retrieve` that generates a .tar.gz but that is corrupted because it was grabbed before the writer was done.

That gets solved if https://github.com/heptio/sonobuoy/pull/584 merges.